keyboards/supplemental/README.md

```diff
@@ -0,0 +1,3 @@
+All of the files in this directory can be thought of as merged together.
```

Comment:
I'm not sure that this makes much sense? It would be helpful to explain here why we have scan code files, and what the differences are between the various files.

Also would note that the scan code definitions exclude 'frame' keys.